### PR TITLE
[ADD] events: added complete documentation

### PR DESCRIPTION
**Pull request by ksc-odoo**
Task ID: [2941079](https://www.odoo.com/web#id=2941079&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

There was no documentation for the Events app, so I created documentation for v14 (which can also be directly applied to v15).

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hey @mivu-odoo 

There was no documentation for Events whatsoever in v14 or v15. Can you please review the content to make sure everything looks okay? EBJ has already looked over the content for accuracy.

Once I get your approval, I'll forward it over to Zac.

Thanks!

**Comment by samueljlieber** [comment]
Hey @mivu-odoo and @ksc-odoo!

I fixed the build errors on this document - just so you are aware, here is what was causing the errors 🙂
1. `events.rst` was not included in the `marketing.rst` `.. toctree::`
2.  Image `:align:` & `:alt:` tags were not properly indented underneath the `.. image::` tag
3. There were a few trailing spaces left throughout the documents
4. On line 17 of `selling_tickets.rst` there was an internal reference to a document that doesn't exist
5. A few `:guilabel:` tags weren't formatted correctly

**Comment by ksc-odoo** [comment]
@StraubCreative Ready for your review!

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

Thanks for the contribution and tagging me in.

For this review I corrected and expanded on the content _only_ in `create.rst` since I saw similar issues that need to be addressed on the other docs. 

Please use the feedback on the change requests below, along with the voluminous feedbacks offered in #2552 and #2534 as a guide to making changes for `selling_tickets.rst` and `track_manage_talks.rst`.

I have already made all/most of the changes locally on my copy of `create.rst` and can push those up with your agreement. Do take the time to review all of the change requests individually since there are requests for rewrites, expansions, and screenshot adjustments for this doc.

Here's an overview of the recurring items to address here on this PR, as well as what's popped up on the last couple PRs:
- **Screenshots are either not in HD or are too large.** Some screenshots are not needed. [Reference.](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#images).
- **Specificity of language:** be specific, always (especially for [headings](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#titles-and-headings)) and avoid vague statements. Be thorough in walking through the UI while providing contextual instruction. Do not use marketing/corporate language in technical documentation.
- **Showing vs. telling:** _show_ the reader how to do things, don't just simply point at things on the UI or feature dump
- **Passive or inconsistent tense:** write in the [simple present tense with an assertive voice.](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#grammatical-tenses) Avoid passive language as much as possible, such as `may` `should` `would` `could` `it is possible` etc. It's okay to tell the reader what to do since that's why they're here: to learn! 😉 
- **Avoid personal pronouns and 2nd-person familiar language**, and stick to third-person instructional. The best way to avoid using people as the subject in sentences is to stick with action-based direction around the UI, e.g., "First, do this...then, do this...lastly...do this"
- **Missing context clues:** remember, documentation readers typically _skim_ the content, so it's important that we constantly remind the reader where we are at any given time. At the beginning of most paragraphs (especially after a lengthy instruction), we can say things like "Now that the questions setting is enabled..." to give them a clue on what just happened, and that, in this case, enabling the setting was necessary to proceed. As well, time order language helps guide the reader and pulls them right into the action on sections that are relevant to them.
- **Wrong or inconsistent tag markup**: use the [RST cheatsheet](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html) as a guide for tagging.
- **Missing opportunities to use call-out tags**, such as `note` `tip` `example` etc. These call-outs help break up walls of text, are a really useful visual aid to bring attention to important details for the reader, and the documentation itself really comes to life with all the colors 🤩 [Reference.](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#alert-blocks-admonitions)

**Comment by StraubCreative** [comment]
Updated partial content for `create.rst` on [8b98d7e](https://github.com/odoo/documentation/pull/2549/commits/8b98d7e13563f4eac356d1ad84084ccc9f8076fe).

Fixed CI errors on [b2c8f69](https://github.com/odoo/documentation/pull/2549/commits/b2c8f69e4d5053f7f02f3e3be41718492de04ea1).

Guidelines on further changes for this PR located on the [original review comment + nested feedbacks here](https://github.com/odoo/documentation/pull/2549#pullrequestreview-1271976282). Outstanding items for `create.rst` were left as unresolved. For comments that are resolved, use these as micro-guides to expand on the broader details listed in the parent review note in order to improve the rest of `create.rst`, as well as `selling_tickets.rst` and `track_manage_talks.rst`.

@ksc-odoo you're good for next round of updates, TIA! 🚀

**Comment by samueljlieber** [comment]
Updates in https://github.com/odoo/documentation/commit/2f71152fbfb6db5ab09c2177f33df7393b41de9f

- fixed build errors
   - removed trailing white spaces
   - fixed a few lines over 100 chars
- added `seealso` tags at the end of each doc
- verified that @StraubCreative's [outstanding items ](https://github.com/odoo/documentation/pull/2549#issuecomment-1409472549) were completed in @ksc-odoo's commit https://github.com/odoo/documentation/commit/c5dc50504eb7efee2d8e258b618f13f2db5abda2 👍

File: content/applications/marketing/events/overview/create.rst
Comment (StraubCreative) [inline content/applications/marketing/events/overview/create.rst:261]:
```suggestion
If the selected :guilabel:`Mailing Type` is guilabel:`SMS, there are four options to send the
invite: :guilabel:`Put in Queue`, :guilabel:`Send Now`, :guilabel:`Schedule`, and :guilabel:`Test`.
```

File: content/applications/marketing/events/overview/create.rst
Comment (StraubCreative) [inline content/applications/marketing/events/overview/create.rst:270]: Suggestion to remove two `guilabel` tags right next to each other.
```suggestion
.. note::
   The :guilabel:`Send Now`, :guilabel:`Schedule`, and :guilabel:`Test` options all function the
   same way for both :guilabel:`Mailing Type` options.
```

File: content/applications/marketing/events/overview/create.rst
Comment (StraubCreative) [inline content/applications/marketing/events/overview/create.rst:290]: Read a couple times and was confusing to me.
Suggested rewrite:

```suggestion
Until an event is published, it will remain hidden from public view on the website and registering
for it will not be possible. To publish an event, navigate to it either from the back end of Odoo
through the :guilabel:`Events` application, or access the hidden event page through the front end as
either a priveliged user or administrator.

If navigating from the back end, go to the event form, and click the :guilabel:`Go to Website` smart
button to reach the event page on the website (on the front end). If starting from the front end,
simply navigate to the event page that needs to be published.

No matter the route, an event page can only be published from the front end. In the upper right
corner of the event page on the website, toggle the switch from the red :guilabel:`Unpublished`
status to the green :guilabel:`Published` status. Doing so instantly makes the event page accessible
to the public on the website.
```
```

File: content/applications/marketing/events/overview/create.rst
Comment (StraubCreative) [inline content/applications/marketing/events/overview/create.rst:294]: The crop here isn't great. Can get a new shot?

```suggestion
.. image:: create/events-frontend-publish.png
   :align: center
   :alt: View of a website page and the option to publish the event in Odoo Events.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo and @samueljlieber 

Thank you for making most of the changes from the last review.
I have another batch, particularly for `create.rst`.

While there was some improvement, we're running into similar some issues as last time:
- screenshots are too large, awkwardly cropped, or not needed
- wording/flow improvements
- precise vs. vague language
- missing or improper use of RST tags (mostly `guilabel`s)

Please address for `create.rst` and reflect the change requests to the other documents as well, if they're needed.

Ideally, with the scope of changes outlined here and other Marketing PRs, I'd like to be able to look at `selling_tickets.rst` and `track_manage_talks.rst` (and their respective image folders) and they'd be more or less perfect across:
- content accuracy and depth
- wording, flow and grammar, and 
- RST formatting (including screenshots) 
 
Give it a shot and lemme know when we're ready for another look, thanks! 😉

File: content/applications/marketing/events/overview/create.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/overview/create.rst:185]: Understood. I created a new image with a more detailed example. However, if it doesn't meet quality standards, feel free to remove this image entirely.

File: content/applications/marketing/events/overview/create.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/overview/create.rst:194]: Since I provided a more detailed sample of this kind of recipient filter this time around (for the suggested image above), I think I can safely omit this image, without confusing the reader.

File: content/applications/marketing/events/overview/create.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/overview/create.rst:215]: Ah, interesting. I was told previously to avoid GIFs, because they're not "responsive" on other devices. But I think a GIF would be best, especially to showcase the speed (and simplicity) of the drag-and-drop action.

Here's a link to a screencast that could (potentially) be used to generate a GIF for this section: https://drive.google.com/file/d/16uIMJWoCNFo6tD1YDVzv3x5b0QxWcUlR/view?usp=sharing

Comment (ksc-odoo) [reply content/applications/marketing/events/overview/create.rst:215]: Here's a fresh screenshot, too (just in case the GIF idea doesn't pan out): https://drive.google.com/file/d/1cxs8nd1NWwihPMVb8BmLqI_A5pwV9-Av/view?usp=sharing

Comment (ksc-odoo) [reply content/applications/marketing/events/overview/create.rst:215]: Feel free to pick either and place it in the doc

**Comment by samueljlieber** [comment]
Hey @ksc-odoo thanks for [image/gif suggestions](https://github.com/odoo/documentation/pull/2549#discussion_r1099277888). I decided to use the gif for this doc, I think it works well!

I also fixed the build errors in the doc 👍

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

Very well done. These were great to read 👍 

I have a number of small suggestions to close us out, mostly related around line breaks and use of parenthesis in the body content.

Please review the suggestions, and indicate if you agree with them, or specify which ones you do not want to include. After that, I can push up all of the revisions and pass to DR.

Again, well done. Looking forward to your reply!

**Comment by ksc-odoo** [comment]
> Hi @ksc-odoo
> 
> Very well done. These were great to read 👍
> 
> I have a number of small suggestions to close us out, mostly related around line breaks and use of parenthesis in the body content.
> 
> Please review the suggestions, and indicate if you agree with them, or specify which ones you do not want to include. After that, I can push up all of the revisions and pass to DR.
> 
> Again, well done. Looking forward to your reply!

@StraubCreative I fully approve all the suggestions, so feel free to push up all of the revisions and pass to DR. Thanks again for all your help with this! It is very much appreciated!

**Review by StraubCreative — APPROVED** [review]
Thanks @ksc-odoo and @samueljlieber for all of your work on this.
Looks great. Passing to DR.

@odoo/doc-review this new documentation on Events app is ready for your review.

File: content/applications/marketing/events.rst
Comment (AntoineVDV) [inline content/applications/marketing/events.rst:16]: Isn't that a bit strange to have a single sub-directory named "Overview"? Couldn't "Events" have "Event essentials", "Sell tickets", and "Track and manage talks" as direct sub-pages?

File: content/applications/marketing/events/overview/create.rst
Comment (AntoineVDV) [inline content/applications/marketing/events/overview/create.rst:112]: Don't impose your own arrow styling to `menuselection` markups.
```suggestion
:menuselection:`Configuration --> Settings` and then enable the :guilabel:`Questions` setting.
```

File: content/applications/marketing/events/overview/create.rst
Comment (AntoineVDV) [inline content/applications/marketing/events/overview/create.rst:143]: Shouldn't that be a tip?

File: content/applications/marketing/events/overview.rst
Comment (AntoineVDV) [inline content/applications/marketing/events/overview.rst:11]: Optional, if you want the file to have the same name as the page title.
```suggestion
   overview/sell_tickets
```
If you do change that, don't forget to rename the folder too.

File: content/applications/marketing/events/overview/track_manage_talks.rst
Comment (AntoineVDV) [inline content/applications/marketing/events/overview/track_manage_talks.rst:78]:
```suggestion
form, or :menuselection:`Reporting --> Attendees` and sorted by event.
```

File: content/applications/marketing/events.rst
Comment (StraubCreative) [reply content/applications/marketing/events.rst:16]: Yes, I agree 👍

File: content/applications/marketing/events/overview.rst
Comment (StraubCreative) [reply content/applications/marketing/events/overview.rst:11]: Will do for all files/folders 👍

**Comment by StraubCreative** [comment]
Addressed latest change requests on [60b04ae](https://github.com/odoo/documentation/pull/2549/commits/60b04aec6e50eaf036b2653624f5108e7702f1e7).

Ready for another look, @AntoineVDV 🙏

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**
